src/platforms/dotnet/profiling/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Set Up Profiling
+description: "Learn how to enable profiling in your app if it is not already set up."
+sidebar_order: 5000
+---
+
+<PlatformContent includePath="profiling/index/preface" />
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments.
+
+<Note>
+
+.NET profiling alpha is available for:
+- .NET 6.0 and above for Windows, Linux and macOS using [.NET Core CLR Diagnostics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/diagnostics-client-library)
+- .NET 6.0 and above for iOS and macCatalyst using native profiling support in [Sentry Apple SDK](/platforms/apple/)
+
+.NET profiling is currently **not** available for:
+- NativeAOT
+- .NET Framework
+- Android
+
+</Note>
+
+## Enable Performance Monitoring
+
+Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, set the `TracesSampleRate` option to the desired value.
+
+```csharp
+using Sentry;
+
+SentrySdk.Init(options =>
+{
+    // ... usual setup options omitted for clarity (see Getting Started) ...
+    
+    // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions.
+    options.TracesSampleRate = 0.1;
+});
+```
+
+## Enable Profiling
+
+To enable profiling, set the `ProfilesSampleRate`. 
+Additionally, for all platforms except iOS/macCatalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.
+
+```csharp {tabTitle:iOS/macCatalyst}
+using Sentry;
+
+SentrySdk.Init(options =>
+{
+    // ... usual setup options omitted for clarity (see Getting Started) ...
+    
+    // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions.
+    options.TracesSampleRate = 0.1;
+    
+    // Sample rate for profiling, applied on top of othe TracesSampleRate, 
+    // e.g. 0.2 means we want to profile 20 % of the captured transactions, i.e. 2 % of all transactions.
+    options.ProfilesSampleRate = 0.2;
+});
+```
```

Comment:
````suggestion
## Enable Profiling

To enable profiling, set the `ProfilesSampleRate`. 
Additionally, for all platforms except iOS/Mac Catalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.

Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, set the `TracesSampleRate` option to the desired value.

```csharp {tabTitle:iOS/macCatalyst}
using Sentry;

SentrySdk.Init(options =>
{
    // ... usual setup options omitted for clarity (see Getting Started) ...
    
    // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions. 1.0 is 100%
    options.TracesSampleRate = 1.0;
    
    // Sample rate for profiling, applied on top of othe TracesSampleRate, 
    // e.g. 0.2 means we want to profile 20 % of the captured transactions, i.e. 2 % of all transactions.
    options.ProfilesSampleRate = 0.2;
});
```
````